tests/common/include/tests/common/check_geometry_navigation.inl

```diff
@@ -183,29 +183,31 @@ struct print_inspector {
     }
 };
 
-// vecmem::host_memory_resource host_mr;
-// auto [d, name_map] = read_from_csv(tml_files, host_mr);
-
-vecmem::host_memory_resource host_mr;
-auto d = create_toy_geometry(host_mr);
-
-// Create the navigator
-using detray_context = decltype(d)::context;
-using detray_track = track<detray_context>;
-/*using detray_inspector = print_inspector<portal_print_inspector,
-                                         surface_print_inspector>;*/
-// using detray_inspector = print_inspector<std::tuple,
-// single_type_print_inspector>;
-using detray_inspector = object_tracer<1>;
-using detray_navigator = single_type_navigator<decltype(d), detray_inspector>;
-using detray_stepper = line_stepper<detray_track>;
-
-detray_navigator n(d);
-detray_stepper s;
-
 // This test runs intersection with all portals of the TrackML detector
 TEST(ALGEBRA_PLUGIN, geometry_discovery) {
 
+    // vecmem::host_memory_resource host_mr;
+    // auto [d, name_map] = read_from_csv(tml_files, host_mr);
+
+    vecmem::host_memory_resource host_mr;
+    auto det_nav = create_toy_geometry(host_mr);
```

Comment:
Do we absolutely need to move the detector to the navigator? I think the detector should not be changed or re-allocated after construction.

Reply:
That's right. Let me change it